packages/apps/plugins/plugin-markdown/src/MarkdownPlugin.tsx

```diff
@@ -30,7 +30,11 @@ export const markdownPlugins = (plugins: Plugin[]): MarkdownPlugin[] => {
   return (plugins as MarkdownPlugin[]).filter((p) => Boolean(p.provides?.markdown));
 };
 
-type MarkdownPluginProvides = SpaceProvides & TranslationsProvides;
+type MarkdownPluginProvides = SpaceProvides &
+  TranslationsProvides & {
+    // todo(thure): Refactor this to be DRY, but avoid circular dependencies. Do we need a package like `plugin-types` 😬? Alternatively, StackPlugin stories could exit its package, but we have no such precedent.
```

Comment:
I think this type will be generalized and not specific to stack. `SpaceProvides` is effectively the same thing. Between interoperable graph and generic actions this should end up not depending on stack but just exposing something that stack can find and use.